src/utilities/util_verify.c

```diff
@@ -62,12 +62,12 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
     int ch;
     char *config, *dump_offsets, *key, *uri;
     bool abort_on_error, do_not_clear_txn_id, dump_address, dump_app_data, dump_blocks, dump_layout,
-      dump_pages, read_corrupt, stable_timestamp;
+      dump_pages, read_corrupt, stable_timestamp, strict;
 
     abort_on_error = do_not_clear_txn_id = dump_address = dump_app_data = dump_blocks =
-      dump_layout = dump_pages = read_corrupt = stable_timestamp = false;
+      dump_layout = dump_pages = read_corrupt = stable_timestamp = strict = false;
     config = dump_offsets = uri = NULL;
-    while ((ch = __wt_getopt(progname, argc, argv, "acd:stu?")) != EOF)
+    while ((ch = __wt_getopt(progname, argc, argv, "acd:eStu?")) != EOF)
```

Comment:
I think this `e` needs to be removed here.

Reply:
Thank you, removed in 9fb9e84d3086327ce1e418ea4f411b6550602b99